### PR TITLE
Add D LSP bundle.

### PR DESCRIPTION
**Pull request by PerilousBooklet**

### CODE REVIEW

**Comment by PerilousBooklet** [comment]
The entry in `lsp`'s `config.lua` will probably need to be optimized following the README of the github repo of `serve-d`.

File: plugins/lsp_d.lua [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+local lspconfig = require "plugins.lsp.config"
+local common = require "core.common"
+local config = require "core.config"
+
+local installed_path = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_d"
+
+lspconfig.serve_d.setup(common.merge({
+  command = { installed_path .. PATHSEP .. "serve-d" .. (PLATFORM == "Windows" and ".exe" or "")}
+}, config.plugins.lsp_tex or {}))
```
Comment (Guldoman) [inline plugins/lsp_d.lua:11]: Woops

File: plugins/lsp_d.lua [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-- mod-version:3 -- lite-xl 2.1
```
Comment (Guldoman) [inline plugins/lsp_d.lua:1]: You can remove the `-- lite-xl 2.1` part. I know it's in some other plugins, but it's not needed.

File: plugins/lsp_d.lua (hunk 1 above)
Comment (PerilousBooklet) [reply plugins/lsp_d.lua:11]: Ah damn

File: plugins/lsp_d.lua (hunk 2 above)
Comment (PerilousBooklet) [reply plugins/lsp_d.lua:1]: Ok

**Comment by PerilousBooklet** [comment]
Conflict should be solved

**Pull request merged**